x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -280,15 +279,7 @@ export const ActionTypeForm = ({
       <EuiSpacer size="xl" />
       {ParamsFieldsComponent ? (
         <EuiErrorBoundary>
-          <Suspense
-            fallback={
-              <EuiFlexGroup justifyContent="center">
-                <EuiFlexItem grow={false}>
-                  <EuiLoadingSpinner size="m" />
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            }
-          >
+          <Suspense fallback={null}>
```

Comment:
Removed the loading spinner here because `ActionForm` shows a loading spinner when loading `ActionTypeForm` causing a double loading spinner when the `ParamsFieldsComponent` is lazy loaded.